moveit_setup_assistant/templates/moveit_config_pkg_template/launch/moveit_rviz.launch

```diff
@@ -4,9 +4,9 @@
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
   <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
 
-  <arg name="config" default="false" />
-  <arg unless="$(arg config)" name="command_args" value="" />
-  <arg     if="$(arg config)" name="command_args" value="-d $(find [GENERATED_PACKAGE_NAME])/launch/moveit.rviz" />
+  <arg name="rviz_config" default="" />
```

Comment:
Renaming this is an API change of launch files, not sure its worth it. If you want to keep it, please update the tutorials 
and add to MIGRATION.md

Reply:
That's a valid point. We could keep the old variable "config" and handle these four cases:
- config == false || config == ''
- config == true
- otherwise, interpret config as file path

Reply:
I'd put it into Migration.md and leave it on master in favor of clear naming. The old argument `config` is only used inside demo.launch and thus I'd not expect any code breakage. The files are only generated for new packages or overwritten, if the user runs setup_assistant explicitly.